saleor/graphql/checkout/tests/test_checkout.py

```diff
@@ -527,6 +527,68 @@ def test_checkout_create_check_lines_quantity_multiple_warehouse(
     assert data["errors"][0]["field"] == "quantity"
 
 
+def test_checkout_create_sets_country_from_shipping_address_country(
+    user_api_client,
+    variant_with_many_stocks_different_shipping_zones,
+    graphql_address_data,
+):
+    variant = variant_with_many_stocks_different_shipping_zones
+    variant_id = graphene.Node.to_global_id("ProductVariant", variant.id)
+    test_email = "test@example.com"
+    shipping_address = graphql_address_data
+    shipping_address["country"] = "US"
+    shipping_address["countryArea"] = "New York"
+    shipping_address["postalCode"] = 10001
+    variables = {
+        "checkoutInput": {
+            "lines": [{"quantity": 1, "variantId": variant_id}],
+            "email": test_email,
+            "shippingAddress": shipping_address,
+        }
+    }
+    assert not Checkout.objects.exists()
+    response = user_api_client.post_graphql(MUTATION_CHECKOUT_CREATE, variables)
+    content = get_graphql_content(response)
+    data = content["data"]["checkoutCreate"]
+    assert data["created"] is True
+    checkout = Checkout.objects.first()
+    assert checkout.country == "US"
+
+
+def test_checkout_create_check_lines_quantity_different_shipping_zone_insufficient_stocks(  # NOQA
```

Comment:
I think we should use the test name longer than the max line length. Maybe we can use a shorter name and add some comment if still need to add more information.

Reply:
If you set `DEFAULT_COUNTRY` in decorator before the test you can remove `different_shipping_zone` from the test name.